### ISSUE TITLE
Redmi 10c Kernel source code

### ISSUE THREAD

**Issue by Mlag123** [MiCode/Xiaomi_Kernel_OpenSource#6623]
Hello to all developers from Xiaomi, I ask you to post the source code of the redmi 10c kernel under the GPL license, and it already exists on redmi 10.
It is very lacking, the stock core is very buggy, and the community wants to change it for the better.

**Comment by Statzar** [comment]
Yes, it would be nice if xiaomi posted the kernel sources for redmi 10C (fog)
